src/components/charts/ChartGroupDetails.tsx

```diff
@@ -131,7 +131,7 @@ export default function ChartGroupDetails() {
                 component={DeleteComponentsName.ChartGroup}
                 redirectTo={true}
                 url={`${URLS.CHARTS}/discover`}
-                reload={false}
+                // reload={false} reload is a function but here boolean value is passed 
```

Comment:
If it's not mandatory then can we remove this completely or pass `noop` to prevent any unnecessary issues